### PR TITLE
Logrotate public chart

### PR DESCRIPTION
**Pull request by ahsan-storm**
Signed-off by: Muhammad Ahsan
username: ahsan-storm
email: ahsanmuhammad1@outlook.com

#### What this PR does / why we need it:
* Creates a public helm chart for logrotate

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by ahsan-storm** [comment]
@linki kindly review this PR, and merge it in master

File: stable/logrotate/.helmignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# OWNERS file for Kubernetes
+OWNERS
```
Comment (glensc) [inline stable/logrotate/.helmignore]: no EOL?

File: stable/logrotate/OWNERS [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- waseem-h
+- rasheedamir
+- ahsan-storm
+reviewers:
+- faizanahmad055
+- kahootali
+- ahmadiq
+- waseem-h
+- rasheedamir
+- ahsan-storm
```
Comment (glensc) [inline stable/logrotate/OWNERS]: no EOL?

**Review by glensc — APPROVED** [review]

File: stable/logrotate/.helmignore (hunk 1 above)
Comment (ahsan-storm) [reply stable/logrotate/.helmignore]: added EOL

File: stable/logrotate/OWNERS (hunk 2 above)
Comment (ahsan-storm) [reply stable/logrotate/OWNERS]: added EOL

File: stable/logrotate/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Logrotate
+
+The logrotate utility is designed to simplify the administration of log files on a system which generates a lot of log files. Logrotate allows for the automatic rotation compression, removal and mailing of log files. Logrotate can be set to handle a log file daily, weekly, monthly or when the log file gets to a certain size.
```
Comment (kdudka) [inline stable/logrotate/README.md]: `rotation compression` --> `rotation, compression` ?

Comment (ahsan-storm) [reply stable/logrotate/README.md]: updated

**Comment by ahsan-storm** [comment]
@linki kindly review this PR, and merge it in master

**Comment by ahsan-storm** [comment]
@linki kindly review this PR, and merge it in master

**Comment by ahsan-storm** [comment]
@linki kindly review this PR, and merge it in master

**Comment by kahootali** [comment]
@linki @davidkarlsen can you please review this PR?

**Comment by linki** [comment]
/assign @davidkarlsen
/unassign @linki

(I'm not a maintainer anymore)

**Comment by nterry** [comment]
@ahsan-storm @davidkarlsen Can we re-open and re-review this? This fulfills a _definite_ need.

**Comment by kahootali** [comment]
@nterry As this was not approved and merged, so we created our own version of the chart and it is hosted at https://github.com/stakater-charts/logrotate

**Comment by nterry** [comment]
@kahootali cheers!

**Pull request closed without merging**